src/xdocs/config_javadoc.xml

```diff
@@ -649,6 +649,13 @@ public int checkReturnTag(final int aTagIndex,
               <td><code>false</code></td>
               <td>6.0</td>
             </tr>
+            <tr>
+              <td>validateParameterOrder</td>
+              <td>Control whether to validate order of <code>param</code> tags.</td>
```

Comment:
We need a config example of new property in use.

Reply:
Alright, will add one.